tests/test_api.py

```diff
@@ -77,6 +79,33 @@ def test_represent(self):
 
         logger.info("✅ representation api test is done")
 
+    def test_represent_encoded(self):
```

Comment:
you may consider to add similar test case for numpy array input

Reply:
Sure, I'll write it till end of the day

Reply:
no problem, you can create a new PR - I am approving this